app/controllers/leads_controller.rb

```diff
@@ -78,8 +67,26 @@ def lead_params
                                 :minority_owned)
   end
 
+  def valid_search?
+    if params[:q1] == "" &&
+      params[:q2] == "" &&
+      params[:q3] == "" &&
+      params[:q4] == "" &&
```

Comment:
Use 4 (not 2) spaces for indenting a condition in an `if` statement spanning multiple lines.